src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/MethodologyStatusPage.tsx

```diff
@@ -159,6 +167,58 @@ const MethodologyStatusPage = () => {
                     Edit status
                   </Button>
                 )}
+
+                {methodologyStatuses && methodologyStatuses.length > 0 && (
+                  <>
+                    <h3>Methodology status history</h3>
+                    <LoadingSpinner
+                      loading={!methodologyStatuses}
+                      text="Loading methodology status history"
+                    >
+                      <table data-testid="methodology-status-history">
+                        <thead>
+                          <tr>
+                            <th scope="col">Date</th>
+                            <th scope="col">Status</th>
+                            <th scope="col">Internal note</th>
+                            <th scope="col">Methodology version</th>
+                            <th scope="col">By user</th>
+                          </tr>
+                        </thead>
+                        {methodologyStatuses && (
```

Comment:
We've already checked that `methodologyStatuses` isn't falsy on line 171 so shouldn't need this check

Reply:
Also removed a similar line in `ReleaseStatusPage.tsx`.